src/ppl/README.md

```diff
@@ -154,78 +95,207 @@ pin placement grid defined with `define_pin_shape_pattern` use:
+place_pins 
+    [-hor_layers h_layers]
+    [-ver_layers v_layers]
+    [-random_seed seed]
+    [-random]
+    [-corner_avoidance length]
+    [-min_distance distance]
+    [-min_distance_in_tracks]
+    [-exclude region]
+    [-group_pins pin_list]
+    [-annealing]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-hor_layers` | The layers to create the metal shapes of pins placed in horizontal tracks. It can be a single layer or a list of layer names. |
+| `-ver_layers` | The layers to create the metal shapes of pins placed in vertical tracks. It can be a single layer or a list of layer names. |
+| `-random_seed` | Refer to the developer arguments [here](#useful-developer-arguments). |
+| `-random` | Refer to the developer arguments [here](#useful-developer-arguments). |
+| `-corner_avoidance` | The distance (in microns) from each corner within which pin placement should be avoided. |
+| `-min_distance` | The minimum distance between pins on the die boundary. This distance can be in microns (default) or in number of tracks between each pin. |
+| `-min_distance_in_tracks` | Flag that allows setting the min distance in number of tracks instead of microns. |
+| `-exclude` | A region where pins cannot be placed. Either `top|bottom|left|right:edge_interval`, which is the edge interval from the selected edge; `begin:end` for begin-end of all edges. |
+| `-group_pins` | A list of pins to be placed together on the die boundary. |
+| `-annealing` | Flag to enable simulated annealing pin placement. |
+
+The `exclude` option syntax is `-exclude edge:interval`. The `edge` values are
+(top|bottom|left|right). The `interval` can be the whole edge, with the `*`
+value, or a range of values. For example, in `place_pins -hor_layers metal2
+-ver_layers metal3 -exclude top:* -exclude right:15-60.5 -exclude left:*-50`
+three intervals are excluded: the whole top edge, the right edge from 15
+microns to 60.5 microns, and the left edge from its beginning to 50 microns.
+
+### Useful developer functions
+
+If you are a developer, you might find these useful. More details can be found in the [source file](./src/IOPlacer.cpp) or the [swig file](./src/IOPlacer.i).
+
+| Function Name | Description |
+| ----- | ----- |
+| `parse_edge` | Parse edge (top/bottom/left/right). |
+| `parse_direction` | Parse direction. |
+| `parse_excludes_arg` | Parse excluded arguments. |
+| `parse_group_pins_arg` | Parse group pins arguments. | 
+| `parse_layer_name` | Parse layer name. |
+| `parse_pin_names` | Parse pin names. |
+| `get_edge_extreme` | Get extremes of edge. |
+| `exclude_intervals` | Set exclude interval. |
+| `add_pins_to_constraint` | Add pins to constrained region. |
+| `add_pins_to_top_layer` | Add pins to top layer. | 
+
+### Useful developer arguments
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-random_seed` | Specify the seed for random operations. |
+| `-random` | When this flag is enabled, the pin placement is random. |
```

Comment:
The same issue as with the other tool, it is not clear which command these flags can be used with.